.gitignore

```diff
@@ -0,0 +1,8 @@
+*.sql
+
+docker/mysql
+docker/app/*/
+!.keep
+
+.env
+!.env.example
```

Comment:
Maybe you could also add `docker-compose.override.yml` here please?

Imho it would be also a good idea to include all files starting with `.env.*` so it would be possible to create a `.env.production` and stay with the upstream